tasks/debian_legacy.yml

```diff
@@ -139,27 +140,14 @@
       when: _frr_version.stdout != frr_version
   when: ansible_lsb.release is version ('18.04', '<')
 
-- name: stat config
+- name: stat defaut config
   stat:
     path: /etc/default/frr
   register: conf_stat
   when:
     - frr_version > "6.0"
 
-- name: Backup old /etc/frr/daemons.conf and /etc/default/frr
-  copy:
-    remote_src: True
-    src: "{{ item }}"
-    dest: "{{ item }}.bak"
-  ignore_errors: true
-  when:
-    - frr_version > "6.0"
-    - conf_stat.stat.exists
```

Comment:
what happens when this file doesn't exist now? I think that's why this was put here originally.

Reply:
/etc/frr/daemons.conf is backed up when changed by config task https://github.com/mrlesmithjr/ansible-frr/blob/master/tasks/config.yml#L4

/etc/default/frr is a file which is not configured by this role, probably even created on package install nowadays. I think in the past it contained some default config params, which conflicted with other /etc/frr/*.conf files, so it was removed by role. The backup is useful only if we use this role on some existing frr deployments which may use that. In that scenario, if something goes bad we would need backup. I doubt that this is the use-case we want to support.

Backup of /etc/frr/*.conf files are more likely to be useful because they are actively used by this role, which still remains covered by another part of code